app/main.py

```diff
@@ -1,8 +1,47 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = list()
+        for car in cars:
+            if not self.clean_power < car.clean_mark:
+                income.append(self.calculate_washing_price(car))
+                self.wash_single_car(car)
+        return round(sum(income), 1)
+
+    def calculate_washing_price(self, car: Any) -> float:
+        price_wash = round(car.comfort_class
+                           * (self.clean_power - car.clean_mark)
+                           * self.average_rating
+                           / self.distance_from_city_center, 1)
+        return price_wash
+
+    def wash_single_car(self, car: Any) -> None:
+        car.clean_mark = self.clean_power
```

Comment:
Due to requirements we need only wash car when self.clean_power < car.clean_mark